docs/user-guide/remapping.ipynb

```diff
@@ -0,0 +1,1325 @@
+{
```

Comment:
<p>Lets change up the order one last time to what we had before. We can move the import block back below the introductory paragraph, like you had before. Can you do the same for the other user guide notebooks as part of this PR too? </p><p> </p><p>Should be </p><p> </p><p>Title</p><p> </p><p>Intro Paragraph</p><p> </p><p>Imports</p><p> </p><p>.. Rest of the nobook </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/UXARRAY/uxarray/pull/807/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>